File: backend/check_products.py
import os
import requests
from dotenv import load_dotenv

load_dotenv()
url = os.environ.get("SUPABASE_URL")
key = os.environ.get("SUPABASE_KEY")

headers = {
    "apikey": key,
    "Authorization": f"Bearer {key}"
}
response = requests.get(f"{url}/rest/v1/products?select=id,name,user_id,collection_id", headers=headers)
print(response.json())

File: backend/check_products.js
const { createClient } = require('@supabase/supabase-js');
require('dotenv').config({ path: '../frontend/.env.local' });
const supabase = createClient(process.env.NEXT_PUBLIC_SUPABASE_URL, process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY);
async function run() {
  const { data } = await supabase.from('products').select('id, name, user_id, collection_id');
  console.log(data);
}
run();
